kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/Labels.java

```diff
@@ -7,14 +7,15 @@
 package io.kroxylicious.kubernetes.operator;
 
 import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
 import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
 
 public class Labels {
 
     public static Map<String, String> standardLabels(KafkaProxy proxy) {
-        HashMap<String, String> labels = new HashMap<>();
+        HashMap<String, String> labels = new LinkedHashMap<>();
```

Comment:
during the refactoring some of the labels changed order in the IT output, I don't know why we haven't seen unit test failures yet but I believe using the unordered hashmap means label order could vary